tests/wasm/variadic.rs

```diff
@@ -37,9 +37,9 @@ extern "C" {
     //                          rest: Vec<String>) -> String;
     #[wasm_bindgen(variadic)]
     fn variadic_sum_rest_vec(first: u8, second: u8, rest: Vec<u8>) -> u8;
-    //#[wasm_bindgen(variadic)]
-    //fn variadic_compare_pairs(first: JsValue, second: JsValue, rest: &[JsValue]);
-    //TODO imported type
+//#[wasm_bindgen(variadic)]
```

Comment:
Looks like this was also touched by `cargo fmt`. Should I revert?

Reply:
But at the same time `cargo fmt` on CI complains about this new formatting :/

I'll try to update my local copy.